microsoft-edge/devtools-guide-chromium/about-tools.md

```diff
@@ -10,11 +10,12 @@ ms.date: 02/02/2022
 # About the list of tools
```

Comment:
In the following files, should the left side of red box be moved to the right to exclude the eye icon? 
Could fix in Paint.
https://github.com/MicrosoftDocs/edge-developer/blob/afc985ef2e1c5a7dabe0e2dfb750d07c980c5cef/microsoft-edge/devtools-guide-chromium/media/console-filter-regex.msft.png
https://github.com/MicrosoftDocs/edge-developer/blob/afc985ef2e1c5a7dabe0e2dfb750d07c980c5cef/microsoft-edge/devtools-guide-chromium/media/console-filter-text.msft.png

Reply:
Done.